README.md

```diff
@@ -1,16 +1,20 @@
-# HTTP-2
+# HTTP-2-Next
```

Comment:
@HoneyryderChuck You'd be best to update this file, with links to the correct/new documentation and usage.